src/phys/dca_step/cluster_solver/ctint/structs/ctint_matrix_configuration.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018 ETH Zurich
+// Copyright (C) 2018 UT-Battelle, LLC
+// All rights reserved.
+//
+// See LICENSE.txt for terms of usage.
+// See CITATION.md for citation guidelines, if DCA++ is used for scientific publications.
+//
+// Authors: Giovanni Balduzzi (gbalduzz@phys.ethz.ch)
+//
+// This file implements ct_int_matrix_configuration.hpp.
+
+#include "dca/phys/dca_step/cluster_solver/ctint/structs/ct_int_matrix_configuration.hpp"
+
+namespace dca {
+namespace phys {
+namespace solver {
+namespace ctint {
+// dca::phys::solver::ctint::
+
+MatrixConfiguration::MatrixConfiguration(const InteractionVertices* H_int, const int bands)
+    : H_int_(H_int), n_bands_(bands), sectors_{Sector(), Sector()} {}
+
+MatrixConfiguration& MatrixConfiguration::operator=(const MatrixConfiguration& rhs) {
```

Comment:
These don't follow the semantics I'd expect for an assignment operator. i.e. H_int_ and n_bands_ are not assigned. Either add those or just use a named function like updateSectors.

Reply:
updated in 727f88b2 with an assertion on the const member.